mhs/inbound/mhs-inbound-env-example.yaml

```diff
@@ -7,6 +7,7 @@ MHS_SYNC_ASYNC_STATE_TABLE_NAME: sync_async_state
 MHS_DYNAMODB_ENDPOINT_URL: http://localhost:8000
 AWS_ACCESS_KEY_ID: test
 AWS_SECRET_ACCESS_KEY: test
+MHS_INBOUND_USE_SSL: true
```

Comment:
I'd set it to `false` - as that's default for OpenTest

Reply:
Wouldn't inbound SSL be enabled by default for OpenTest?

Reply:
I asked on the team channel and guys say it should be disabled, @mattd-kainos

Reply:
@bartek-sarul confirmed it is enabled, I will  remove to previous version then

Reply:
MHS_INBOUND_USE_SSL is only for testing purpose to enable request being sent directly to inbound without ssl